app/com/linkedin/drelephant/spark/heuristics/ConfigurationHeuristic.scala

```diff
@@ -128,18 +135,29 @@ object ConfigurationHeuristic {
     lazy val serializer: Option[String] = getProperty(SPARK_SERIALIZER_KEY)
 
     lazy val serializerSeverity: Severity = serializer match {
-      case None => Severity.NONE
+      case None => Severity.MODERATE
       case Some(`serializerIfNonNullRecommendation`) => Severity.NONE
```

Comment:
I don't know if that's true, otherwise the unit tests will fail. I think it all depends on the value of foo. I tried the code locally, and got the following outputs:

````
scala> val foo = 50
foo: Int = 50

scala>     Some(42) match {
     |       case Some(`foo`) => "first" 
     |       case Some(_) => "second"
     |     }
res1: String = second


scala> val foo = 42
foo: Int = 42

scala>     Some(42) match {
     |       case Some(`foo`) => "first"
     |       case Some(_) => "second"
     |     }
res2: String = first

```

I am going to leave this code unchanged.